taiga_halo2/src/circuit/vp_examples/token.rs

```diff
@@ -381,6 +381,7 @@ pub fn generate_input_token_note_proving_info<R: RngCore>(
     InputNoteProvingInfo::new(
         input_note,
         merkle_path,
+        None,
```

Comment:
So of this is `None`, then it is calculated by the `new` constructor?

Reply:
Maybe the argument for anchor in the `new` method should be called `custom_anchor`, and if no custom anchor is provided then the standard one is calculated. Just to make it a bit easier to understand for the user.